images/kubevirt/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
```

Comment:
This comment is not true. However, assuming that there will be more work to do in other parts of the installer repo, you probably want to make it true by specifying the kubevirt approvers and reviewers in the OWNERS_ALIASES file rather than here.